recognition/include/pcl/recognition/hv/hv_go.h

```diff
@@ -45,6 +45,8 @@ namespace pcl
           int id_;
       };
 
+      typedef boost::shared_ptr<RecognitionModel> RecognitionModelPtr;
```

Comment:
Any reason to break convention here? Usually this would be defined inside the `RecognitionModel` as `Ptr`.

Reply:
If we typedef it inside `RecognitionModel` then it becomes a dependent name and needs to be qualified with `typename`. So on every usage we would need to spell `typename RecognitionModel::Ptr` instead of `RecognitionModelPtr`. Given that this all is a private business (both `struct` and `typedef`), I did not think there is a binding convention to follow. But if you prefer more verbose version, I'm can commit an update.

Reply:
Meh. No real strong opinion. We can leave it as is.